pyogrio/raw.py

```diff
@@ -17,7 +18,8 @@
 
 
 def read(
-    path,
+    path_or_buffer,
+    /,
```

Comment:
Is this a typo?

Reply:
No, this is what actually makes the first argument a positional-only argument. See https://peps.python.org/pep-0570/